package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "api-sidecar",
+  "version": "1.0.0",
+  "description": "CENNZnet storage index and HTTP service",
+  "main": "src/index.js",
+  "repository": "ssh://git@github.com/cennznet/api-sidecar",
+  "author": "developers@centrality.ai",
+  "license": "Apache",
+  "scripts": {
+    "api": "ts-node src/api/index.ts",
+    "nft:stats": "ts-node src/scanner/scanNFTStats.ts",
+    "nft:finalizationHead": "ts-node src/scanner/trackfinalizeHead.ts"
```

Comment:
can we make the script names to be `start:api`, `start:scanner` and `start:head-tracker` respectively?

Reply:
and add `"format": "prettier --config .prettierrc.json ./ --write"` then run `yarn format` to prettify your codebase